xbmc/windowing/X11/GLContextEGL.cpp

```diff
@@ -321,7 +329,9 @@ EGLConfig CGLContextEGL::getEGLConfig(EGLDisplay eglDisplay, XVisualInfo *vInfo)
     }
     if (value == (EGLint)vInfo->visualid) {
       eglConfig = eglConfigs[i];
-      break;
+      //Don't select the first one, let's keep the last instead
+      //eg. SGX 545 on X11 returns 3 configurations with the same visualid but the first has depth=0
```

Comment:
You are right, my bad. This should have not been included in the commit. It was the dirtier from the two solutions for my platform. The clean one https://github.com/xbmc/xbmc/pull/8234 (saving eglconfig) is already integrated in this version, so these modifications are not needed. Should I commit on the HEAD or change the previous commit?